BSHUF/config/cmake/ConfigureChecks.cmake

```diff
@@ -17,7 +17,7 @@ if (APPLE)
   list (LENGTH CMAKE_OSX_ARCHITECTURES ARCH_LENGTH)
   if (ARCH_LENGTH GREATER 1)
     set (CMAKE_OSX_ARCHITECTURES "" CACHE STRING "" FORCE)
-    message(FATAL_ERROR "Building Universal Binaries on OS X is NOT supported by the H5BSHUF project. This is"
+    message (FATAL_ERROR "Building Universal Binaries on OS X is NOT supported by the H5BZ2 project. This is"
```

Comment:
Is this change intended in a BSHUF subdirectory?

Reply:
no

Reply:
fixed